compiler-rt/lib/interception/interception_win.cpp

```diff
@@ -462,7 +462,7 @@ static size_t GetInstructionSize(uptr address, size_t* rel_offset = nullptr) {
   return 4;
 #endif
 
-#if SANITIZER_WINDOWS64
+#if SANITIZER_WINDOWSx64
```

Comment:
Kindly add a comment on why this is x64 only.

Reply:
`SANITIZER_WINDOWS64` means `SANITIZER_WINDOWS_x64` in this case and on line 552 because those are windows64 cases that parse x86_64 assembly.